src/vaadin-login-overlay-wrapper.html

```diff
@@ -15,28 +15,6 @@
         outline: none;
       }
 
-      [part="card"] {
```

Comment:
is this change necessary after all? consider reverting it, so that PR does not change neither `src` not Lumo